source/common/config/filter_json.cc

```diff
@@ -184,6 +184,11 @@ void FilterJson::translateHttpConnectionManager(
          json_tracing->getStringArray("request_headers_for_tags", true)) {
       tracing->add_request_headers_for_tags(header);
     }
+
+    tracing->mutable_client_sampling()->set_value(json_tracing->getDouble("client_sampling", 100));
```

Comment:
We no longer allow v1 config changes. Please remove this as well as the schema change in the other file.